content/en/blog/_posts/2019-9-17-SIG-Scalability

```diff
@@ -0,0 +1,26 @@
+SIG Scalability: Learning Kubernetes the Hard Way
+By: Alex Handy
+
+There is a strain of thinking in the software development community, that pushes for the most difficult and rigorous possible method of learning a new language or system. These tend to go by the moniker of “Learn __ the Hard Way.” Examples abound: Learn Code the Hard Way, Learn Python the Hard Way, and of course the one that really started the whole movement, Zed Shaw’s courses in the topic.
+
+While there are folks out there who will offer you a “Learn Kubernetes the Hard Way” type experience--most notably Google’s Kelsey Hightower. That particular example is tied to one cloud in particular, however, and in the spirit of this thing, any “Hard Way” project should probably at least attempt to cover every aspect of the platform being taught.
+
+Therefore, the real way to “Learn Kubernetes the Hard Way,” is to join the CNCF and get involved in the project itself. And there is only one SIG that could truly offer a full stack learning experience for Kubernetes: SIG Scalability.
+
+The team behind SIG Scalability is responsible for detecting and dealing with issues that arrise when Kubernetes clusters are working with upwards of a thousand nodes. In fact, said Wojiciech Tyczynski, staff software engineer at Google and member of SIG Scalability, the standard size for a test cluster for this SIG is over 5,000 nodes.
+
+And yet, this SIG is not composed of Ph.D’s in highly scalable systems designs. Many of the folks working with Tyczynski, for example, joined the SIG knowing very little about these types of issues, and often, very little about Kubernetes.
```

Comment:
Nit: This is a notable call-out, but it might flow better if you refactor it earlier into the blog post.